widget/radio.go

```diff
@@ -1,12 +1,11 @@
 package widget
 
 import (
-	"image/color"
```

Comment:
This order of imports was correct. Update your IDE to use the goimport format

Reply:
can you take a screenshot of the Goland settings that control this?  I have no idea what combination to use to get the correct result.  I thought we had it ironed out after the icon colorization PR, I haven't changed any settings, it's still broken.  I fixed it manually this time, pushed.